service/common/build.gradle.kts

```
@@ -83,6 +83,9 @@ dependencies {
 
   implementation(platform(libs.azuresdk.bom))
   implementation("com.azure:azure-core")
+
+  implementation(platform(libs.micrometer.bom))
```

Nit: the platform can be compileOnly as well
```suggestion
  compileOnly(platform(libs.micrometer.bom))
```